content/docs/forms.md

```diff
@@ -178,33 +193,34 @@ class FlavorForm extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/JbbEzX?editors=0010)
+[**Probeer Het Op CodePen**](https://codepen.io/gaearon/pen/JbbEzX?editors=0010)
 
-Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
+In het algemeen zorgt dit ervoor dat `<input type="text">`, `<textarea>`, en `<select>` allemaal op een zeer vergelijkbare manier functioneren - ze accepteren allemaal een `value` attribuut dat je kan gebruiken om een gecontroleerd component te implementeren.
 
-> Note
+> Opmerking
 >
-> You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag:
+> Je kan een array doorgeven aan het `value` attribuut, wat je toelaat om meerdere opties te selecteren in een `select` tag:
 >
 >```js
 ><select multiple={true} value={['B', 'C']}>
 >```
 
-## The file input Tag {#the-file-input-tag}
+## De file input Tag {#the-file-input-tag}
 
-In HTML, an `<input type="file">` lets the user choose one or more files from their device storage to be uploaded to a server or manipulated by JavaScript via the [File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications).
+In HTML, laat een `<input type="file">` de gebruiker één of meerdere bestanden kiezen uit de opslag van hun apparaat om geüpload te worden naar een server of gemanipuleerd te worden door JavaScript via de [File API](https://developer.mozilla.org/nl/docs/Web/API/File/Using_files_from_web_applications).
 
 ```html
 <input type="file" />
 ```
 
-Because its value is read-only, it is an **uncontrolled** component in React. It is discussed together with other uncontrolled components [later in the documentation](/docs/uncontrolled-components.html#the-file-input-tag).
+Omdat zijn waarde read-only is, is het een **ongecontroleerd** component in React.
```

Comment:
```suggestion
Omdat zijn waarde alleen-lezen is, is het een **on-aangestuurd** component in React.
```

Reply:
But I would probably _want_ to say `kan je het component niet aansturen in React`